LanguageChangedEvent.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
I can't find this extra blank line in PSR-2 and the example there also don't use it. Can you explain?